docs/src/pages/docs/en/migration/from-0.x-to-1.0.md

```diff
@@ -342,3 +345,21 @@ Also, for any custom themes you may have created, as a result of moving from `ke
   </template>
 </template>
 ```
+
+## Using the new `icon` slot
+
+All components with a single icon now expose a slot called `icon` instead of a different named slot for each component. e.g:
```

Comment:
suggestion(non-blocking): Consider adding a simple list here of all of the single element slot renames, similar to https://github.com/muxinc/media-chrome/blob/main/docs/src/pages/docs/en/migration/from-0.x-to-1.0.md?plain=1#L165-L169)